src/components/forms/apply-form.tsx

```diff
@@ -219,7 +231,67 @@ const ApplyForm = () => {
                         })}
                     />
                 </div>
-                <div className="tw-mb-7.5">
+                <Checkbox
+                    label="Have you previously attended any coding bootcamps or tech education programs?"
+                    id="hasAttendedPreviousCourse"
+                    {...register("hasAttendedPreviousCourse")}
+                />
+                <br />
+                {watchHasAttendedPreviousCourses && (
+                    <motion.div
+                        layout
+                        className="tw-mb-7.5"
+                        initial={{ opacity: 0, scale: 0.4 }}
+                        animate={{ opacity: 1, scale: 1 }}
+                    >
+                        <label htmlFor="previousCourses" className="tw-text-heading tw-text-md">
+                            List previous courses *
+                        </label>
+                        <TextArea
+                            id="previousCourses"
+                            placeholder="Fundamentals of Spouse Stealing"
```

Comment:
LOL we should probably change this placeholder message. I cackled though.

Reply:
Updated the placeholders with the most recent commit